Configurations.md

```diff
@@ -2481,6 +2481,34 @@ fn main() {
 
 See also [`max_width`](#max_width) and [`use_small_heuristics`](#use_small_heuristics)
 
+## `single_line_simple_if`
+
+Allows simple single expression if blocks to format on one line. Useful in the case of keeping `let-else` guards format consistent with `if` guards.
+
+Note that line will still break if:
+1. The condition length is over the `max_width`/`single_line_simple_if_max_width length`
+2. The block contains a trailing semicolon
+3. The block contains a single or multi-lined comment
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
+- **Stable**: No
+
+#### `false` (default):
```

Comment:
we'll need some documented examples of what formatting the option produces when it's `true` or `false`.